content/concepts/refs.mdx

```diff
@@ -2,241 +2,123 @@ import { Aside } from "~/components/configurable/Aside";
 
 # Refs
 
-Refs, or references, allow you to have direct access to DOM elements. 
+Refs, or references, are a special attribute that can be attached to any element, and are used to reference a DOM element or a component instance.
+They are particularly useful when you need to access the DOM nodes directly or invoke methods on a component.
 
-Refs in Solid are assignments that occur at creation time before elements are attached to the 
-document DOM. This lets you access its properties and methods directly, allowing for manipulatinos 
-such as animations and event handling.
+## Accessing DOM elements
 
-To use `ref`, you simply need to declare a variable and use it as a ref attribute:
+One way of accessing DOM elements is through [element selectors](https://developer.mozilla.org/en-US/docs/Web/API/Document_object_model/Locating_DOM_elements_using_selectors) such as [`document.querySelector`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) or [`document.getElementById`](https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById).
+Since elements in Solid can be added or removed from the DOM based on state, however, you need to wait until the element is attached to the DOM before accessing it.
+This can be done by using [`onMount`](/reference/onMount) to wait until the element is attached to the DOM before accessing it:
 
 ```tsx
-let myDiv;
```

Comment:
@LadyBluenotes for some reason GH won't let me do multiline comments 🙃 This is in regards to the simple example and the aside note.

I think including the simple usage example and the note at the top like this is good. IMO docs should be laid out to give you the gist of what it is and why it is useful as quickly as possible and then more detailed sections that you can decided to dig into or not for more info. 

If this is brought down then you have to going searching to find it, I think the idea of what it is takes longer to get to and understand.

The typescript note about definitive assignment assertion is just important IMO because everyone who uses typescript will quickly run into that problem and I think it qualifies as necessary for basic usage in typescript.

Reply:
Bringing the TypeScript aside down with the information that is relevant to it provides both value and context to the user. It doesn't make sense to put something at the top of the file just because it's important, especially if you lose out on why it is mentioned.